### PR TITLE
Fix: Fixed an issue where folder icon customization apply before saving

### LINKED ISSUE #11917: Bug: Folder customization are applied directly
### Description

The folder customizations are applied directly and the window minimizes as soon as you select a new icon or click Restore Default.
The Save & Restore buttons have no effect on the behavior of the customizations page.

### Steps To Reproduce

1. Select a folder
2. Open the properties
3. Navigate to Customizations
4. Select a new icon or click Restore default

### Requirements

- Properties window is not minimized and remains open until Save or Cancel is clicked
- The save button saves the new icon
- Cancel button discards the changes
- If possible, "Restore Default" should highlight the new icon in the selection after it is clicked

### Files Version

2.4.59.0

### Windows Version

10.0.25324.1011

### Log file

n/a

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

Fix an issue which has been reported.

### Motivation and Context

- Do not change the icons when 'Restore' button clicked
- Do not change the icons when an icon selected

### PR Checklist
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11917
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app
   2. Open properties of a folder
   3. Switch tab to Customization
   4. Change icons

### Screenshots

_No response_

### CODE REVIEW

**Review by ferrariofilippo — APPROVED** [review]
LGTM and works for me!

**Comment by 0x5bfa** [comment]
I don't know. I thoughts so well but when I write private fields/properties as lower, IntelliSense suggestions have a ‘Name Violation’ item.

**Comment by yaira2** [comment]
> Just a question: shouldn't private variables names start with a lowercase letter?

Yes

**Comment by 0x5bfa** [comment]
By original functionality, if you switch tabs, props window will discard user changes unless we don't have Apply button or fix that issue.

File: src/Files.App/ViewModels/Properties/CustomizationViewModel.cs [hunk 1]
```diff
@@ -1,125 +1,82 @@
...
-using System.Collections.ObjectModel;
 using System.IO;
-using System.Threading.Tasks;
 using Windows.Storage.Pickers;
 using Microsoft.UI.Windowing;
 
 namespace Files.App.ViewModels.Properties
 {
+	/// <summary>
+	/// Represents ViewModel for Files.App.CustomizationPage
```
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/ViewModels/Properties/CustomizationViewModel.cs]: Fixed

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**